common/vm/vm.go

```diff
@@ -33,9 +33,24 @@ func NewGenericVM(
 
 	readyFunc := func(r *command.Runner) (*remote.Command, error) { return command.WaitForCloudInit(ctx, r) }
 	if os.GetType() == commonos.WindowsType {
-		// On Windows, there is no equivalent of cloud init, but the code wait until ssh connection is ready
-		// so it is OK to not have a ready function.
-		readyFunc = nil
+		cmd := `for ($i = 0; $i -le 120; $i++) { 
```

Comment:
Are you using `120` to wait for 2 minutes ?

Reply:
Yes